scripts/post_install/CMakeLists.txt

```diff
@@ -2,5 +2,6 @@ set(KRATOS_GENERATE_PYTHON_STUBS OFF CACHE BOOL "Generate hints for binary pytho
 
 if (${KRATOS_GENERATE_PYTHON_STUBS})
     install(CODE "message(STATUS \"Trying to install python stub files to enable python hinting support for cpp libraries in IDEs...\")")
-    install(CODE "execute_process(COMMAND \"${PYTHON_EXECUTABLE}\" \"${CMAKE_CURRENT_SOURCE_DIR}/stub_generation.py\" \"${CMAKE_INSTALL_PREFIX}\" --quiet)")
+    cmake_path(CONVERT ${PYTHON_EXECUTABLE} TO_CMAKE_PATH_LIST CONVERTED_PY_EXEC)
+    install(CODE "execute_process(COMMAND \"${CONVERTED_PY_EXEC}\" \"${CMAKE_CURRENT_SOURCE_DIR}/stub_generation.py\" \"${CMAKE_INSTALL_PREFIX}\" --quiet)")
```

Comment:
```CMake
\"${CMAKE_CURRENT_SOURCE_DIR}/stub_generation.py\"
```

Most likely the problem is the `/` in the 2nd argument of the install (The one I quoted). I assume the same solution that @avdg81 proposed will work for the other arguments. 

In the meantime @ddiezrod you can just set `KRATOS_GENERATE_PYTHON_STUBS=OFF` I think. It is not a critical feature....